### ISSUE TITLE
You can’t open SwiftDefaultApps preferences pane because it doesn’t work on an Intel-based Mac.

### ISSUE THREAD

**Issue by glyph** [Lord-Kamina/SwiftDefaultApps#3]
In the GUI:

![screen shot 2017-12-20 at 7 19 22 pm](https://user-images.githubusercontent.com/716529/34239221-b6f10f96-e5ba-11e7-86f6-8ccad4da71f0.png)

In Console.app:

```
default	19:19:10.072813 -0800	System Preferences	[NSPrefPaneBundle instantiatePrefPaneObject] (/Users/glyph/Library/PreferencePanes/SwiftDefaultApps.prefPane): principalClass is nil.
```

```
$ sw_vers
ProductName:	Mac OS X
ProductVersion:	10.13.2
BuildVersion:	17C88
```

**Comment by d4nirod** [comment]
Same here. Also on MacOS 10.13.2

**Comment by mattpangaro** [comment]
Build in Xcode results in the same error for me on 10.13.3

**Comment by Lord-Kamina** [comment]
This sucks so much. Listen, I'm still on a 2011 mac running Sierra. I'm not dropping this yet but I'll put it on hold until April when I plan to get a new MBP.

**Comment by glyph** [comment]
No worries. Thanks for making this, and hope you enjoy that new macbook :).

**Comment by quinncomendant** [comment]
@Lord-Kamina, can you please post an update to [this Stack Overflow question](https://superuser.com/questions/548119/how-do-i-configure-custom-url-handlers-on-os-x/644593#644593) when you update SDA to work on Intel? Thanks!

**Comment by Lord-Kamina** [comment]
I've just released v1.1.1 although it's just a maintenance release. Nonetheless, this should now be fixed, owing largely to somnebody getting their shit together in the 10.13.4 release; can you please confirm?

**Comment by glyph** [comment]
I built my own since the released binary is unsigned, but, it works now, and previously it didn't :).

Thanks for getting it fixed!
